alembic/versions/2024_05_22_18e3b2aba252_add_yield_columns.py

```diff
@@ -0,0 +1,84 @@
+"""2024_05_22_add_yield_columns
+
+Revision ID: 18e3b2aba252
+Revises: 5fd7e8758fb1
+Create Date: 2024-05-22 14:01:15.197675
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "18e3b2aba252"
+down_revision = "5fd7e8758fb1"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+
+    # rename table IlluminaSampleRunMetrics to IlluminaSampleSequencingMetrics
+    op.rename_table(
+        old_table_name="illumina_sample_run_metrics",
+        new_table_name="illumina_sample_sequencing_metrics",
+    )
+    # add column yield
+    op.add_column(
+        table_name="illumina_sample_sequencing_metrics",
+        column=sa.Column("yield", sa.Integer(), nullable=True),
+    )
+    # add column yield_q30
+    op.add_column(
+        table_name="illumina_sample_sequencing_metrics",
+        column=sa.Column("yield_q30", sa.Integer(), nullable=True),
```

Comment:
yield_q30 is defined as float in the model (Num6_2)

Reply:
fixed